backend-zs/controllers/stories.controller.js

```diff
@@ -0,0 +1,14 @@
+var Story = require('../models/story.model').Story;
+
+exports.getStories = function (_req, res) {
+    Story.find()
+        .select("title author shortDescription numberLikes category")
+        .then(stories => { res.send(res.json(stories)) })
```

Comment:
why are you using both `res.send()` and `res.json()` ? I think using one of them is enough. Using `res.json()` seems good to be consistent with the other team.